sunkit_image/utils/tests/test_utils.py

```diff
@@ -17,6 +17,7 @@
 
 @pytest.fixture
 def smap():
+    import sunpy.data.sample
```

Comment:
the horror

Reply:
Yeah I want to check something but the tox env wont install on my local machine.

Reply:
Doesn't work. I can't see to get the sample data downloaded for the test. I have to come up with another method.